pkg/webhooks/topic_validator.geso

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package webhook
+package webhooks
```

Comment:
**Question**: Why was this file's extension changed from `.go` to `.geso`? Was that intentional? I can see that some of the errors have been moved, had this meant to be a working copy which was supposed to be removed?